cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/integration_utils_test.go

```diff
@@ -0,0 +1,337 @@
+/*
+Copyright © 2021 VMware
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	plugins "github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen/core/plugins/v1alpha1"
+	fluxplugin "github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen/plugins/fluxv2/packages/v1alpha1"
+	"google.golang.org/grpc"
+	kubecorev1 "k8s.io/api/core/v1"
+	kuberbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func checkEnv(t *testing.T) fluxplugin.FluxV2PackagesServiceClient {
+	enableEnvVar := os.Getenv(envVarFluxIntegrationTests)
+	runTests := false
+	if enableEnvVar != "" {
+		var err error
+		runTests, err = strconv.ParseBool(enableEnvVar)
+		if err != nil {
+			t.Fatalf("%+v", err)
+		}
+	}
+
+	if !runTests {
+		t.Skipf("skipping flux plugin integration tests as %q not set to be true", envVarFluxIntegrationTests)
+	} else {
+		if up, err := isLocalKindClusterUp(t); err != nil || !up {
+			t.Fatalf("Failed to find local kind cluster due to: [%v]", err)
+		}
+		var fluxPluginClient fluxplugin.FluxV2PackagesServiceClient
+		var err error
+		if fluxPluginClient, err = getFluxPluginClient(t); err != nil {
+			t.Fatalf("Failed to get fluxv2 plugin due to: [%v]", err)
+		}
+		rand.Seed(time.Now().UnixNano())
+		return fluxPluginClient
+	}
+	return nil
+}
```

Comment:
Thanks - yes, now it explicitly fails if someone tries to enable them but hasn't got the kind cluster running etc.